modules/client/src/internal/client/methods.ts

```diff
@@ -866,4 +867,14 @@ export class ClientMethods extends ClientCore implements IClientMethods {
     }
     return toPluginRepo(pluginRepo, releaseMetadata, buildMetadata);
   }
+  public async getProtocolVersion(
```

Comment:
I imagined this to be a general function that you can apply to all OSx contracts (as mentioned here: https://discord.com/channels/672466989217873929/1132983258347487315/1132983261115723857).

We added the `protocolVersion` number function in v1.3.0.
This means that all prior contracts must be `1.0.0`.

In my tests, I used this (which you might want to improve).
```ts
export async function getProtocolVersion(
  contract: Contract
): Promise<[number, number, number]> {
  let protocolVersion: [number, number, number];
  try {
    contract.interface.getFunction('protocolVersion');
    protocolVersion = await contract.protocolVersion();
  } catch (error) {
    if (error.code === errors.INVALID_ARGUMENT) {
      protocolVersion = [1, 0, 0];
    } else {
      throw error;
    }
  }
  return protocolVersion;
}
```